### PR TITLE
charts: modify the keyword #75

### PR DESCRIPTION
**Pull request by runkecheng**
modified:   charts/Chart.yaml
        modified:   charts/README.md
        modified:   charts/templates/_helpers.tpl
        modified:   charts/templates/configmap.yaml
        modified:   charts/templates/rbac.yaml
        modified:   charts/templates/secrets.yaml
        modified:   charts/templates/service.yaml
        modified:   charts/templates/serviceaccount.yaml
        modified:   charts/templates/servicemonitor.yaml
        modified:   charts/templates/statefulset.yaml
        modified:   charts/values.yaml
        modified:   charts/Chart.yaml
        modified:   docs/zh-cn/charts.md

rename the `xenondb` to `radondb-mysql`

### CODE REVIEW

File: charts/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
-name: xenondb
-description: XenonDB is a High-Availability MySQL cluster, open source.
-home: https://github.com/radondb/xenondb.git
+name: radondb-mysql
+description: radondb-mysql is a High-Availability MySQL cluster, open source.
```
Comment (zhyass) [inline charts/Chart.yaml:3]: Radondb-mysql

File: docs/zh-cn/charts.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# XenonDB Charts
+# radondb-mysql Charts
```
Comment (zhyass) [inline docs/zh-cn/charts.md:1]: RadonDB-MySQL

File: docs/zh-cn/charts.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
 ## 简介
 
-XenonDB chart 使用 [Helm](https://helm.sh) 包管理器在 [Kubernetes](http://kubernetes.io) 集群上部署一主多从 [MySQL](https://MySQL.org) 集群。
+radondb-mysql chart 使用 [Helm](https://helm.sh) 包管理器在 [Kubernetes](http://kubernetes.io) 集群上部署一主多从 [MySQL](https://MySQL.org) 集群。
```
Comment (zhyass) [inline docs/zh-cn/charts.md:5]: RadonDB-MySQL

**Review by zhyass — CHANGES_REQUESTED** [review]

File: charts/README.md [hunk 4]
```diff
@@ -49,16 +52,16 @@ $ helm delete my-release
...
+kubectl delete pvc data-my-release-radondb-mysql-1
+kubectl delete pvc data-my-release-radondb-mysql-2
 ```
 
 The commands remove all the Kubernetes components associated with the chart and deletes the release completely.
 
 # Configuration
 
-The following table lists the configurable parameters of the xenondb chart and their default values.
+The following table lists the configurable parameters of the radondb-mysql chart and their default values.
```
Comment (zhyass) [inline charts/README.md:64]: RadonDB-MySQL

File: charts/README.md [hunk 5]
```diff
@@ -1,10 +1,13 @@
 # RadonDB-MySQL
+
 RadonDB-MySQL is a open-source, cloud-native, High-Availability cluster solutions that is based on MySQL.
```
Comment (hustjieke) [inline charts/README.md:3]: `a open-source`--->`an open-source`
`solutions that is based on MySQL`--->`solutions  based on MySQL`

**Review by zhyass — CHANGES_REQUESTED** [review]

**Comment by zhyass** [comment]
LGTM

**Review by zhyass — APPROVED** [review]

**Pull request merged**